client/pages/user-registration/registration.template.html.slim

```diff
@@ -51,22 +51,14 @@
         |  You will create an organization now as part of the registration process.
         |  You will be the administrator of this organization and may invite other members later.
 
-    .col-sm-6.col-xs-12
-      .form-group
-        label for='organization-display-name' Name
-        input#organization-display-name.form-control type='string' maxlength=50 ng-model='organization.displayName' required=true
-        p.help-block
-          | The name of the organization.
-
-    .col-sm-6.col-xs-12
-      .form-group ng-class='{"has-error": registrationForm.name.$dirty && registrationForm.name.$invalid || organizationNamePlaceholderTaken}'
+    .col-sm-12.col-xs-12
```

Comment:
See what it looks like with a `.col-sm-6` here and for the checkbox. Keep your preferred version (I don't care which).

Either use two `.col-sm-6.col-xs-12` or two `.col-xs-12`, but `.col-sm-12.col-xs-12` is redundant.